webapp/javascript/pages/ContinuousSingleView.tsx

```diff
@@ -114,6 +115,19 @@ function ContinuousSingleView() {
     if (!isAnnotationsEnabled) {
       return null;
     }
+
+    if (props.value) {
```

Comment:
I don't understand this, when `props.value` exist we show annotation info? What is `props.value`? What does `ContextMenu` have to do with `AnnotationInfo`?

Reply:
not relevant, I removed this stuff